pkgs/by-name/we/wezterm/package.nix

```diff
@@ -137,33 +117,13 @@ rustPlatform.buildRustPackage rec {
 
   passthru = {
     # the headless variant is useful when deploying wezterm's mux server on remote severs
-    headless = rustPlatform.buildRustPackage {
-      pname = "${pname}-headless";
+    headless = import ./headless.nix {
```

Comment:
Is this cross friendly? Looks suspicious to me

Reply:
Should be equivalent, I am not using `callPackage` or anything.